README.md

```diff
@@ -89,7 +95,9 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD is a tool we use to visualize and describe the data relating to the major entities that will exist in our programs.
+A Lamp can have many Genies - One to many
+A Person can have many pets and pet can have more than one owner (married couple or married couple with children) - Many to Many  
 ```
```

Comment:
:+1:  Note the relationships can change depending on the application. For example: One genie could belong to one lamp, or a person has many pets, a pet has one owner.